hermit-cli/crates/hermitfile-parser/src/main.rs

```diff
@@ -207,48 +202,48 @@ fn parse_hermit_args() -> HermitCliArgs {
 
     let mut args_iter = args.iter();
     args_iter.next();
-    loop {
-        let arg = match args_iter.next() {
-            Some(arg) => arg,
-            None => break,
-        };
-        if arg == "-o" {
-            output_path = match args_iter.next() {
-                Some(arg) => arg.clone(),
-                None => panic!("-o provided without file"),
-            };
-            continue;
-        } else if arg == "-f" {
-            hermitfile_path = match args_iter.next() {
-                Some(arg) => arg.clone(),
-                None => panic!("-o provided without file"),
-            };
-            continue;
-        } else if arg == "-h" || arg == "--help" {
-            print!(
-                "{}",
-                r#"hermit.com [-f <path_to_Hermitfile] [-o <output_path>]
-
-If a path to a `Hermitfile` is not provided, it tries to load `Hermitfile` from
-the current directory. If an `output_path` is not provided, the hermit is
-written to `wasm.com` in the currently directly. On Unix-like operating systems
-you must `chmod +x wasm.com` to make it executable. This is required because
-WASI does not have a `chmod` function.
-"#
-            );
-            std::process::exit(0);
-        } else if arg.len() >= 4 {
-            let arg_bytes = arg.as_bytes();
-            let (name, value) = arg_bytes.split_at(3);
-            if name == b"-o=" {
-                output_path = std::ffi::OsStr::from_bytes(&value).into();
-                continue;
-            } else if name == b"-f=" {
-                hermitfile_path = std::ffi::OsStr::from_bytes(&value).into();
-                continue;
+    while let Some(arg) = args_iter.next() {
+        match (arg.len(),arg.to_str()) {
+            (_,Some("-o")) => {
+                if let Some(output_arg) = args_iter.next() {
+                    output_path = output_arg.clone();
+                } else {
+                    panic!("-o provided without file");
+                }
+            }
+            (_,Some("-f")) => {
+                if let Some(hermitfile_arg) = args_iter.next() {
+                    hermitfile_path = hermitfile_arg.clone();
+                } else {
+                    panic!("-f provided without file");
+                }
+            }
+            (_,Some("-h") | Some("--help")) => {
+                print!(
+                    "{}",
+                    r#"hermit.com [-f <path_to_Hermitfile] [-o <output_path>]
+    
+    If a path to a `Hermitfile` is not provided, it tries to load `Hermitfile` from
```

Comment:
These lines being intended looks off to me when I run `--help`. Please remove the indentation change from the help output.

Reply:
done, `cargo fmt` saved the day on this one.
Maybe we can add a check for formatting in ci, to avoid this type of issue in future.